js/templates/settings.html

```diff
@@ -0,0 +1,463 @@
+                        </div>
+                    </div>
+
+                    <div class="flexCol-9 ">
+                        <div class="fieldItem padding0">
+                            <div class="fieldItem-selectWrapper noIcon">
+                                <textarea name="refund_policy" id="refund_policy" rows="6" class="fieldItem-textarea" value=""><%= ob.user.refund_policy%></textarea>
+                            </div>
+                        </div>
+                    </div>
+
+                </div>
+            </div>
+            <div class="flexCol-12 flex-border hide js-tabTarg js-blocked">
+                <div class="flexRow">
+                    <div class="flexRow custCol-border-secondary">
+                        <div class="txtFieldWrapper txtFieldWrapper-bar searchWrapper">
+                            <input type="text"
+                                   name="blocked"
+                                   id="blocked"
+                                   class="txtField txtField-bar search"
+                                   placeholder="<%= polyglot.t('SearchForFollowersPlaceholder') %>"  />
+                        </div>
+                    </div>
+                    <div class="flexRow js-list1"></div>
+                </div>
+            </div>
+            <div class="flexCol-12 flex-border hide js-tabTarg js-advanced">
+                <div class="flexRow ">
+
+                    <div class="flexCol-3  borderRight ">
+                        <div class="fieldItem">
+                            <label for="notifications">
+                                <%= polyglot.t('Enable notifications') %>
+                            </label>
+                        </div>
+                    </div>
+
+                    <div class="flexCol-9 ">
+                        <div class="fieldItem padding0">
+                            <div class="fieldItem-selectWrapper noIcon">
+                                <input type="checkbox" name="notifications" id="notifications"
+                                    <% if(ob.user.notifications) { %>
+                                    checked="checked"
+                                    <% } %>
+                                value="true">
+                            </div>
+                        </div>
+                    </div>
+
+                </div>
+                <div class="flexRow ">
+
+                    <div class="flexCol-3  borderRight ">
+                        <div class="fieldItem">
+                            <label for="ssl">
+                                <%= polyglot.t('Enable SSL') %>
+                            </label>
+                        </div>
+                    </div>
+
+                    <div class="flexCol-9 ">
+                        <div class="fieldItem padding0">
+                            <div class="fieldItem-selectWrapper noIcon">
+                                <input type="checkbox" name="ssl" id="ssl"
```

Comment:
Add the fieldItem class to the input, and add a label after it (which gets the :before that creates the styled checkbox).